src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -0,0 +1,165 @@
+        ['highFrequency'] = 3700000000
+
+    # Load device info
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+
+    # Move devices within 150km of FSS
+    device_1['installationParam']['latitude'] = \
+        fss['deploymentParam'][0]['installationParam']['latitude'] + 0.05
+    device_1['installationParam']['longitude'] = \
+        fss['deploymentParam'][0]['installationParam']['longitude'] + 0.05
+    device_2['installationParam']['latitude'] = \
+        fss['deploymentParam'][0]['installationParam']['latitude'] + 0.12
+    device_2['installationParam']['longitude'] = \
+        fss['deploymentParam'][0]['installationParam']['longitude'] + 0.05
+    device_3['installationParam']['latitude'] = \
+        fss['deploymentParam'][0]['installationParam']['latitude'] + 0.15
+    device_3['installationParam']['longitude'] = \
+        fss['deploymentParam'][0]['installationParam']['longitude'] + 0.05
+
+    # Move GWBL within 150km of FSS
+    gwbl['record']['deploymentParam'][0]['installationParam']['latitude'] = \
+        fss['deploymentParam'][0]['installationParam']['latitude'] + 0.1
+    gwbl['record']['deploymentParam'][0]['installationParam']['longitude'] = \
+        fss['deploymentParam'][0]['installationParam']['longitude']
+
+    # Create info for grants
+    grant_1 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_1['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_1['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_2 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_2['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_2['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_3 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_3['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_3['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+
+    # Creating conditionals for Cat B device
+    self.assertEqual(device_1['cbsdCategory'], 'B')
+    conditionalParameters = {
+        'cbsdCategory': device_1['cbsdCategory'],
+        'fccId': device_1['fccId'],
+        'cbsdSerialNumber': device_1['cbsdSerialNumber'],
+        'airInterface': device_1['airInterface'],
+        'installationParam': device_1['installationParam'],
+        'measCapability': device_1['measCapability']
+    }
+    del device_1['cbsdCategory']
+    del device_1['airInterface']
+    del device_1['installationParam']
+
+    # Create the actual config.
+    conditionals = {
```

Comment:
conditionalRegistrationData changes coming up in: PR #479, so here too, conditionals need to be just a list of registration requests. And we wont need the checks in Line 140.